app/src/main/java/org/p2p/wallet/striga/wallet/interactor/StrigaWalletInteractor.kt

```diff
@@ -76,6 +78,19 @@ class StrigaWalletInteractor(
         )
     }
 
+    suspend fun initEurOffRamp(eurAmount: BigDecimal): StrigaInitEurOffRampDetails {
```

Comment:
@eduardmaximovich

Reply:
Could you please add special exception if there's no banking details? I would use it to navigate from confirm cell to withdraw screen
Something like this:
```
suspend fun getEurBankingDetailsOrThrow(): StrigaUserBankingDetails {
        val details = walletRepository.getUserBankingDetails(getEurAccountId()).unwrap()

        if (details.bankingIban.isNullOrBlank() || details.bankingBic.isNullOrBlank()) {
            throw StrigaNoBankingDetailsProvided()
        }

        return details
    }

@Throws(StrigaNoBankingDetailsProvided::class)
suspend fun initEurOffRamp(eurAmount: BigDecimal): StrigaInitEurOffRampDetails {
   val bankingDetails = getEurBankingDetailsOrThrow()

   return withdrawalsRepository.initEurOffRamp(
            sourceAccountId = eurAccountId,
            amount = eurAmount,
            iban = bankingDetails.iban,
            bic = bankingDetails.bic
        ).unwrap()
}
```